internal/kube/deployment.go

```diff
@@ -13,7 +13,7 @@ func GetDeployment(namespace string, name string) (*v1.Deployment, error) {
 		return nil, err
 	}
 
-	deployment, err := client.AppsV1().Deployments(namespace).Get(context.TODO(), name, meta_v1.GetOptions{})
+	deployment, err := client.AppsV1().Deployments(namespace).Get(context.TODO(), name, metav1.GetOptions{})
```

Comment:
Currently, deployment is only available from apps/v1 api, but this is possible from other api if kubernetes version is less than 1.16

I think it would be better if klocust only supports in the environment with >= kubernetes 1.16 in README.md file

Reply:
This is not an issue if users do not change the api version in the manifest file :)

Reply:
Hm, we need to use v1beta for old k8s versions for users at some where. :)
And if v1beta will be deprecated, then we will change to v1.

Reply:
FYI, v1beta1 is deprecated in kubernetes 1.16. 
https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/

Then are you planning to support v1 and v1beta1 at the same time??

Reply:
then, I think it's good klocust support v1 only.
If user want use v1beta, user can change default templates in `${user home}/.klocust/_default_templates`.

Reply:
Is it possible to apply v1beta1 with AppsV1()?

Reply:
Oh, It's not. If we want to support two versions then we have to change that part.
(divide by obj.GetVersion())
At now, I think more than v1.16 is enough.